src/domain/Paginators/Pagination/pagination.ts

```diff
@@ -0,0 +1 @@
+// @ts-ignore
```

Comment:
what lol

Reply:
Oh I see this file is to hack styleguide into showing examples

Reply:
Yeah, having an empty ts file causes rollup typescript plugin to throw an error. This is to mute that.